assets/src/edit-story/app/media/mediaProvider.js

```diff
@@ -53,6 +53,9 @@ function MediaProvider({ children }) {
     removeProcessing,
     updateMediaElement,
   } = actions;
+  const {
```

Comment:
Is this related to fix?

Reply:
Sort of. Tests failed otherwise because a function was used before initialization. Not sure why it works right now.

Reply:
Actually, unrelated. Storybook is currently broken: https://google.github.io/web-stories-wp/?path=/story/playground-stories-editor--default

Will fix separately.

Reply:
Hmm looks like it was already fixed though. But the storybook wasn't deployed.